src/firefly/java/edu/caltech/ipac/firefly/server/ws/WsServerCommands.java

```diff
@@ -174,18 +176,32 @@ private static String getResultResponse(WsResponse wsResp) throws Exception {
         /**
          * If table, it will get it and then put it into WS under relative path passed to {@link WsServerParams}
          *
-         * @param sp
-         * @return
+         * @param sp parameters
+         * @return response as JSON string
          * @throws Exception
          */
         public String doCommand(SrvParam sp) throws Exception {
 
             TableServerRequest request = sp.getTableServerRequest();
             if (request == null) throw new IllegalArgumentException("Invalid/Missing table request");
-            FileInfo f = new SearchManager().getFileInfo(request);
+
+            TableUtil.Format tblFormat = sp.getTableFormat();
+            String fileNameExt = tblFormat.getFileNameExt();
+
+            File file = File.createTempFile(request.getRequestId(), fileNameExt, ServerContext.getTempWorkDir());
+
+            SearchManager am = new SearchManager();
+            try (OutputStream out = new BufferedOutputStream(new FileOutputStream(file), (int) (32 * FileUtil.K))) {
```

Comment:
so the file size is limited to be 32K?

Reply:
No, it's the size of the buffer to speed up writing the file.